app/models/event.rb

```diff
@@ -22,4 +22,10 @@ class Event < ApplicationRecord
   def service_description
     service_category.service_category_name
   end
+
+  def exception_notes(zip_code)
+    return unless (event_zip = event_zip_codes.find_by(zip_code: zip_code))
```

Comment:
Is this just a little bit of logic that only puts the exception not in if the zip code it matches the searched for zip code.